attributes/default.rb

```diff
@@ -2,4 +2,4 @@
 default['osl-git']['secrets_item']    = 'item'
 default['osl-git']['secrets']['credentials'] = []
 
-default['osl-git']['ius_git_package'] = 'git224'
+default['osl-git']['ius_git_package'] = 'git226'
```

Comment:
I might have typoed this, it needs to be `git236`

Reply:
Ah okay just pushed the changes to reflect the changes